velox/expression/Expr.h

```diff
@@ -722,6 +724,12 @@ class ExprSet {
       core::ExecCtx* execCtx,
       bool enableConstantFolding = true);
 
+  ExprSet(
```

Comment:
It seems these 2 are no longer needed with delete expr separated from remaining filter expr